includes/api/class-give-api.php

```diff
@@ -1735,6 +1735,10 @@ public function output( $status_code = 200 ) {
 	function user_key_field( $user ) {
 
 		if ( ( give_get_option( 'api_allow_user_keys', false ) || current_user_can( 'manage_give_settings' ) ) && current_user_can( 'edit_user', $user->ID ) ) {
+
+			// Flush Rewrite Rules to avoid 404 error using Give API for first time.
+			flush_rewrite_rules();
```

Comment:
@DevinWalker I've implemented the required change and simplified the usage of `generate_api_key` fn to achieve the functionality from single fn. Also, tested the case from `User Profile` as well as `Tools > API` and checked Generate API, Regenerate API and Revoke API.